tests/unit/preprocessor/_time/test_time.py

```diff
@@ -496,7 +496,98 @@ def test_time_rms(self):
         expected = np.array([(5 / 3)**0.5], dtype=np.float32)
         assert_array_equal(result.data, expected)
 
+    # @pytest.mark.parametrize("percentile,exp", )
+    def test_time_percentile(self):
+        """Test for time percentile of a 1D field."""
+
+        # For some reason I can't use pytest parametrize. If I'm being stupid,
```

Comment:
@ESMValGroup/tech-reviewers can I get a hand please on this: why can't I use `pytest.mark.parametrize` around these test class methods? Is it because we're subclassing `unittest.TestCase`? 

If I try to use parametrize like so

```python
@pytest.mark.parametrize("percentile,exp", (
        (10, [0.1, 2.2, 5]),
        (50, [0.5, 3, 5]),
        (90, [0.9, 3.8, 5]),
    ))
def test_season_percentile_uneven(self, percentile, exp):
```

I get the following error

```python
...
def _callTestMethod(self, method):
>       method()
E       TypeError: test_season_percentile_uneven() missing 2 required positional arguments: 'percentile' and 'exp'
```

Reply:
Moved question to #1616, apologies for the ping